data-processor/src/test/groovy/io/micronaut/data/processor/visitors/FindSpec.groovy

```diff
@@ -438,7 +436,7 @@ interface TestRepository extends CrudRepository<Book, Long> {
         when:
             def method = repository.findPossibleMethods("findTop30OrderByTitle").findFirst().get()
         then:
-            method.stringValue(Query).get() == 'SELECT book_."id",book_."author_id",book_."genre_id",book_."title",book_."total_pages",book_."publisher_id",book_."last_updated" FROM "book" book_ ORDER BY book_."title" ASC'
+            method.stringValue(Query).get() == 'SELECT book_."id",book_."author_id",book_."genre_id",book_."title",book_."total_pages",book_."publisher_id",book_."last_updated" FROM "book" book_'
```

Comment:
Where has the 'ORDER by title` gone for the `findTop30OrderByTitle`?

Reply:
The sort is dynamic so it cannot be in the query